Communitheme/gtk-3.0/_common.scss

```diff
@@ -4501,7 +4500,7 @@ decoration {
   // see bug #722563
   $_wm_border: transparentize(black, 0.77);
 
-  box-shadow: 0 3px 9px 1px transparentize(black, 0.5);
+  box-shadow: $window_decoration_shadow;
```

Comment:
So are we changing the window decoration shadow as well?

I am not sure about this, I was actually thinking about increasing windows shadow. Ambiance has bigger shadows, also other nice looking systems (OSX) have way bigger shadows. That would improve also the separation between backdropped windows when are overlapping (and not tiled)